src/modules/FFmpeg/FFDecSW.cpp

```diff
@@ -339,8 +339,6 @@ bool FFDecSW::open(StreamInfo &streamInfo, VideoWriter *)
 		return false;
 	if (codec_ctx->codec_type == AVMEDIA_TYPE_VIDEO)
 	{
-		if (codec->capabilities & CODEC_CAP_DR1)
```

Comment:
As I checked with FFmpeg 2.2 the CODEC_FLAG_EMU_EDGE flag was always set. Maybe I'm missing something?